app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostController < ApplicationController
+
+def index
+  @posts = Post.all
+end
+
+def show
+  @posts = Post.all
```

Comment:
The purpose of the `show` page is to display information about a single post. Here you're retrieving **all** of the posts in your database.